src/Sylius/Bundle/CoreBundle/MessageDispatcher/ResetPasswordDispatcher.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\MessageDispatcher;
+
+use Sylius\Bundle\CoreBundle\Message\Admin\Account\ResetPassword;
+use Symfony\Component\Messenger\MessageBusInterface;
+
+/** @experimental */
```

Comment:
Why experimental 🤔?

Reply:
Well, it touches only parts that are experimental, so why shouldn't it be?

Reply:
I'm wondering, should the `ResetPassword` class still be experimental? 🤔 It was, as it was the part of ApiBundle, which is still experimental by default. But it's used now not only in API but also in UI, which is quite stable I think 💃 So I would vote for removing it here and there - functionality is not experimental for sure, only it's usage in the API 🖖